Tests/GitHubTests/GitHubTests.swift

```diff
@@ -1,4 +1,6 @@
+@testable import AsyncHTTPClient
```

Comment:
Apparently HTTPClient requests are completely opaque, so in order to snapshot test them I'm doing a testable import to get access to the underlying value.

We might just want to fully abstract around the idea of an HTTP client in the future to avoid this. More work and kinda annoying, but it seems like this library isn't currently built with testability/modularity in mind.